src/linear_algebra/matrix/adjugate.lean

```diff
@@ -151,6 +152,18 @@ calc ∑ x in s, cramer A (λ j, f j x) i
 ... = cramer A (λ (j : n), ∑ x in s, f j x) i :
   by { rw [sum_cramer, cramer_apply], congr' with j, apply finset.sum_apply }
 
+lemma cramer_submatrix_equiv (A : matrix m m α) (e : n ≃ m) (b : n → α) :
+  cramer (A.submatrix e e) b = cramer A (b ∘ e.symm) ∘ e :=
```

Comment:
simp?

Reply:
The RHS doesn't seem obviously simpler to me.

Reply:
It is simpler in the sense that `cramer` is applied to a simpler matrix.

Reply:
`cramer` is basically unused, so I don't think it's worth spending time bikeshedding simp lemmas. note that `cramer_smul` is also not marked simp, but the RHS has a "simpler" matrix.